doc/tutorials/amf.md

```diff
@@ -137,7 +137,8 @@ int main()
   NMFALSFactorizer nmf;
   mat W, H;
   mat V = randu<mat>(100, 100);
-  double residue = nmf.Apply(V, W, H);
+  size_t r = 90;
```

Comment:
90 is a very large rank; I'd suggest some more like 5 or 10 which is likely to be more realistic for many problems.